korman/properties/modifiers/avatar.py

```diff
@@ -111,6 +121,11 @@ class PlasmaSittingBehavior(idprops.IDPropObjectMixin, PlasmaModifierProperties,
     facing_degrees = IntProperty(name="Tolerance",
                                  description="How far away we will tolerate the avatar facing the clickable",
                                  min=-180, max=180, default=45)
+    sitting_cam = PointerProperty(name="Camera (optional)",
```

Comment:
I think I would prefer for this to be `sitting_camera` instead.

Reply:
Done!